test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1047,6 +1192,129 @@ def test_cgroup_reserve_mem(self):
         self.assertEqual(vmem_resv.value, 97280)
         self.assertEqual(vmem_resv.unit, 'kb')
 
+    def test_cgroup_multi_node(self):
+        """
+        Test that multi-node jobs with cgroups works fine
+        """
+        name = 'CGROUP16'
+        self.load_config(self.cfg1 % ("", "", "", ""))
+        a = {'Resource_List.select': '2:ncpus=1:mem=300mb',
+             'Resource_List.place': 'scatter', ATTR_N: name}
+        j = Job(TEST_USER, attrs=a)
+        j.create_script(self.sleep15_job)
+        jid = self.server.submit(j)
+        a = {'job_state': 'R'}
+        self.server.expect(JOB, a, jid)
+        cpath = "/sys/fs/cgroup/memory/pbspro/" + jid
+        rv1 = self.du.isdir(hostname=self.momA, path=cpath, sudo=True)
+        self.assertEqual(rv1, True)
+        rv2 = self.du.isdir(hostname=self.momB, path=cpath, sudo=True)
+        self.assertEqual(rv2, True)
```

Comment:
would assertTrue() or assertFalse() be enough here rather than assertEqual ()

Reply:
May be I am not sure. Any specific reason against using assertEqual?

Reply:
Just a convention, that I see,  used in other places of the file

Reply:
While they are technically the same, I see the purpose of assertTrue() is to indeed assert something is true.  I think it is the right call to use here.